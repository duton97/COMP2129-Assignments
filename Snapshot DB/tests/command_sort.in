SET a 4 3 5 2 1
SORT a
GET a
SORT b
BYE
