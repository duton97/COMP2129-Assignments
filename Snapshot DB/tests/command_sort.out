> ok

> ok

> [1 2 3 4 5]

> no such key

> bye
